californium-core/src/main/java/org/eclipse/californium/core/network/RemoteEndpoint.java

```diff
@@ -197,7 +199,11 @@ public void setEstimatorValues(long rto, long rtt, long rttvar, int estimatorTyp
 	public Queue<Exchange> getNonConfirmableQueue(){
 		return nonConfirmableQueue;
 	}
-	
+
+	public ReliabilityLayerParameters getConfigForEndpoint() {
+		return endpointReliabilityLayerParameters;
+	}
```

Comment:
"ForEndpoint" is not needed ? and I find getConfig to general ?

Why not something like getReliabilityLayerParameters() ?

by the way the specficiation seems to call that : "Transmission Parameters"
(see https://tools.ietf.org/html/rfc7252#section-4.8 )